iOS/airbnb/FooterTable/FooterTableViewCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FooterTableViewCell.swift
+//  airbnb
+//
+//  Created by 박혜원 on 2021/05/27.
+//
+
+import UIKit
+
+class FooterTableViewCell: UITableViewCell, IdentityInfo {
+
+    var title: UILabel = {
+        let label = UILabel()
+        label.text = "title"
+        label.font = UIFont.systemFont(ofSize: 17)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    var content: UILabel = {
+        let label = UILabel()
+        label.text = "content"
+        label.font = UIFont.systemFont(ofSize: 17)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configure()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        configure()
+    }
+    func configure() {
```

Comment:
> 함수 형식 

```suggestion
    }
    
    func configure() {
```
=> 이렇게 맞추는 것 제안드립니다.